Remotion/Web/Core/Res/Themes/NovaViso/HTML/TreeView.css

```diff
@@ -1,697 +1,112 @@
-  height: 16px;
-  width: 16px;
-}
-
-/* Applied to the node's head. */
-span.treeViewNodeHead
-{
-}
-
-/* Applied to the node's head if it is the selected node. */
-span.treeViewNodeHeadSelected
-{
-}
-
-span.treeViewNodeHead,
-span.treeViewNodeHeadSelected
-{
-  display: block;
-  height: 1.7em;
-}
-
-/* Nodes with an expander icon */
-span.treeViewNode > a + span.treeViewNodeHead,
-span.treeViewNode > a + span.treeViewNodeHeadSelected
-{
-  margin-left: 19px;
-}
-
-span.treeViewNodeHead a,
-span.treeViewNodeHeadSelected a
-{
-  vertical-align: -0.4em;
-
-  padding: 2px;
-  /* Synchronized with focus/hover */
-  padding-top: 3px;
-  padding-bottom: 3px;
-
-  color: Black;
+  text-decoration: underline;
 }
 
-li:focus > span.treeViewNode span.treeViewNodeHead a,
-span.treeViewNodeHead a:hover,
-li:focus > span.treeViewNode span.treeViewNodeHeadSelected a,
-span.treeViewNodeHeadSelected a:hover
+ul.treeViewRoot span.treeViewNode:hover span.treeViewNodeHead:before,
+ul.treeViewRoot span.treeViewNode span.treeViewNodeHeadSelected:before,
+ul.treeViewRoot li:focus > span.treeViewNode span.treeViewNodeHead:before,
+ul.treeViewRoot li:focus > span.treeViewNode span.treeViewNodeHeadSelected:before
 {
-  padding: 1px;
-  /* Ensures that diacritic marks can be rendered without being cut off while not chaning the overall height of the node. */
-  padding-top: 2px;
-  padding-bottom: 2px;
-
-  border: 1px solid #7CB2E6;
-  background-color: #E5F0FC;
-  outline: none;
+  background-color: var(--color-highlight);
 }
 
-span.treeViewNodeHeadSelected a:focus
+ul.treeViewRoot span.treeViewNodeHead:before,
+ul.treeViewRoot span.treeViewNodeHeadSelected:before
 {
-  outline: none;
+  content: " ";
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  bottom: 0;
+  z-index: var(--z-index-hidden);
 }
 
-li:focus > span.treeViewNode span.treeViewNodeHead a > span,
-li:focus > span.treeViewNode span.treeViewNodeHead a:hover > span,
-li:focus > span.treeViewNode span.treeViewNodeHeadSelected a > span
+ul.treeViewRoot span.treeViewNode,
+ul.treeViewRoot span.treeViewNodeHead > a,
+ul.treeViewRoot span.treeViewNodeHeadSelected > a
 {
-  text-decoration: underline;
+  display: flex;
+  align-items: center;
+  padding-top: var(--remotion-themed-spacing-half);
+  padding-bottom: var(--remotion-themed-spacing-half);
 }
 
-span.treeViewNodeHead a:active,
-span.treeViewNodeHeadSelected a,
-span.treeViewNodeHeadSelected a:active
+ul.treeViewRoot:not(.whitespaceNoWrap) span.treeViewNodeHead,
+ul.treeViewRoot:not(.whitespaceNoWrap) span.treeViewNodeHeadSelected
 {
-  padding: 1px;
-  /* Synchronized with focus/hover */
-  padding-top: 2px;
-  padding-bottom: 2px;
-
-  border: 1px solid #569CE0;
-  background-color: #C5E0FC;
+  --current-padding: calc(var(--nesting-depth) * var(--treeview-padding));
+  --chevron-size: calc(var(--treeview-chevron-size) + var(--treeview-chevron-gap));
+  width: calc(var(--treeview-width) - var(--current-padding) - var(--chevron-size));
```

Comment:
Hmm...the standard usecase is to set the --treeview-width to 100% so I don't think we actually need to have a variable for this. Reasoning: 100% is evaluated here based on the parent and unless I'm mistaken, using calc(100%) here will either never work or always work. Either way, we should be able to drop the css variable for treeview-width. Is that correct?

Reply:
The point of the variable is to limit the width of the text node, resulting in the word wrapping behavior from before. But I see your point, if the width is set to something like 100%, or limited by the parent width without an explicit width set, this calculation would not work. It was the only solution I found to restore the old wrapping behavior, while working with the new "highlight the whole line" design.

Reply:
Meh, okay. Then we skip this bit, just add the nowordwrap css class to the root and apply disable the wordwrap. What's the behavior then? It will break at the end of the longest line, correct? Please create a bug ticket with "wordwrap behavior is undefined" and we already have an open question for this anyway so we keep it simple for now.

Reply:
> What's the behavior then? It will break at the end of the longest line, correct?

yes. I'll add a ticket.